scan-irb/report-en.tex

```diff
@@ -58,7 +58,23 @@
 \begin{description}[font=\normalfont,align=left,labelwidth=12em]
 \item [\textbf{\VAR{pat_name|e}}]
 \item [\textbf{Date of Birth}] \VAR{birth_date|e}
-\item [\textbf{Specimen}] Midnasal, self-collected
+\item [\textbf{Specimen}]
+  %- if swab_type == "mts"
+  Midnasal,
+  %- elif swab_type == "ans"
+  Anterior,
+  %- else
+  \errmessage{Invalid swab_type "\VAR{swab_type}"}
```

Comment:
TIL about `\errmessage`, nice! I clearly didn't know about it when I did my divide-by-zero hack elsewhere in this doc. I assume you tested and it works as expected?

Reply:
Yes it does. Thank you TeX stack exchange for all of my LaTeX knowledge. 

Should I use `\errmessage` to replace the divide-by-zero code you wrote?

Reply:
> Should I use `\errmessage` to replace the divide-by-zero code you wrote?

Sure!

Reply:
I'll do this.. eventually!